whisper/stt/processing/streaming.py

```diff
@@ -43,17 +44,20 @@ async def wssDecode(ws: WebSocketServerProtocol, model_and_alignementmodel):
     try:
         config = json.loads(res)["config"]
         sample_rate = config["sample_rate"]
+        language = config.get("language", None)
```

Comment:
Don't we need to update the doc, specifying the language can be specified in the configuration of the streaming ?

Reply:
Also, why not to support language="`*`" (as in the env variable for the language in offline decoding)
that would need to replace "*" by None when reading the config

Reply:
> Don't we need to update the doc, specifying the language can be specified in the configuration of the streaming request?

Yes I need to update the doc